source/_posts/2025-01-03-3-2-1-backup.markdown

```diff
@@ -0,0 +1,71 @@
+
+## Backups have never been this easy
+
+<img src='/images/blog/2025-01-backups/backups.png' style='border: 0;box-shadow: none;' alt="the backup page">
+
+Home Assistant backups are incredibly good, backing up every setting and add-on, making restores feel a bit like magic. It even works across different CPU architectures! Up until recently setting up a 3-2-1 backup strategy in Home Assistant was not entirely straightforward. This changes with Home Assistant 2025.1, where we're elevating our backup experience with three new important features.
+
+- **Automatic encrypted backups** - You can now schedule automatic backups every night or once a week, ensuring you always have a recent backup at hand.
+- **Automatic cleanup** - No longer worry about backups overrunning your disk space, as you can now set the maximum number of backups that should be stored.
+- **Backup locations** - We have added a new system for integrations to provide backup locations (also known as [backup agents](https://developers.home-assistant.io/docs/core/platform/backup)), allowing you to sync backups to both local or off-site storage. It's launching with support for [network shares](https://my.home-assistant.io/redirect/storage/) and [Home Assistant Cloud](https://my.home-assistant.io/redirect/cloud/) (included at no additional cost to subscribers), with more locations to follow.
+
+<!--more-->
+
+### Get started with our Backup Wizard
+
+<img src='/images/blog/2025-01-backups/wizard.png' style='border: 0;box-shadow: none;' alt="the backup wizard">
+
+The new automatic backup feature in Home Assistant 2025.1 comes with a new easy to use backup wizard. With a couple of clicks, you can backup your Home Assistant system on a regular schedule and easily choose the places where backups are stored. New users will be prompted with a wizard to make sure they set up automatic encrypted backups. Selecting the recommended settings ensures that a backup is made every night, that three backups are kept, and that they are uploaded to all available locations.
+
+### Backup locations
+
+<img src='/images/blog/2025-01-backups/locations.png' style='border: 0;box-shadow: none;' alt="backup locations">
+
+At launch only a few backup locations will be available, but we've made it easy for developers to integrate more, so expect a lot of options to follow. As part of this renewed focus on smart home resilience set out in [our roadmap](/blog/2024/06/12/roadmap-2024h1/#next-priority-encrypted-cloud-backup), cloud backup has been added to [Home Assistant Cloud](/cloud/) , which will be available day one to all subscribers for no additional cost. It will only store the latest backup, which must be under 5GB (you'll be notified to repair it if it is too large to upload). If you lose your other backups, you can always go to [your Cloud account page](https://account.nabucasa.com/) and download the latest backup.
```

Comment:
```suggestion
At launch, only a few backup locations will be available, but we've made it easy for developers to integrate more, so expect a lot of options to follow. As part of this renewed focus on smart home resilience set out in [our roadmap](/blog/2024/06/12/roadmap-2024h1/#next-priority-encrypted-cloud-backup), cloud backup has been added to [Home Assistant Cloud](/cloud/), which will be available day one to all subscribers for no additional cost. It will only store the latest backup, which must be under 5GB (you'll be notified to repair it if it is too large to upload). If you lose your other backups, you can always go to [your Cloud account page](https://account.nabucasa.com/) and download the latest backup.
```